src/hotspot/cpu/riscv/templateInterpreterGenerator_riscv.cpp

```diff
@@ -526,7 +526,7 @@ address TemplateInterpreterGenerator::generate_safept_entry_for(TosState state,
   address entry = __ pc();
   __ push(state);
   __ call_VM(noreg, runtime_entry);
-  __ fence(0xf, 0xf);
+  __ membar(MacroAssembler::AnyAny);
```

Comment:
This does not look like a "redundant non-null assertion" :)

Reply:
OK. I think I can propose this change in a seperate PR. I will see if I can find other similar occurences.
See updated change. Thanks.